roles/kubernetes/node/tasks/haproxy.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: haproxy | Write static pod
+  template:
+    src: manifests/haproxy.manifest.j2
```

Comment:
There is a risk to end up with both the haproxy and the nginx manifest and switching the `loadbalancer_apiserver_type`?